packages/@ember/-internals/glimmer/lib/components/input.ts

```diff
@@ -57,6 +57,19 @@ import InternalComponent from './internal';
   <Input @value={{this.searchWord}} @enter={{this.query}} />
   ```
 
+  Starting with Ember Octane, we recommend using the {{on}} modifier to call actions 
+  on specific events, such as the input event.
+
+  <label for="input-name">Name:</label>
+  <Input
+    @id="input-name"
+    @value={{this.name}}
+    {{on "input" this.validateName}}
+  />
```

Comment:
Can you wrap this in triple backticks? That will help it render like a code block.

Reply:
Sure thing! What about the linting error in the checks?